extensions/integration-tests/src/utils.ts

```diff
@@ -137,10 +138,63 @@ export async function assertFileGenerationResult(filepath: string, retryCount: n
 	while (retryCount > 0 && !exists) {
 		--retryCount;
 		exists = fs.existsSync(filepath);
-		await sleep(5000);
+
+		if (!exists) {
+			await sleep(5000);
+		}
 	}
 
 	assert(exists, `File ${filepath} is expected to be present`);
 	assert(fs.readFileSync(filepath).byteLength > 0, 'File should not be empty');
 	fs.unlinkSync(filepath);
-}
\ No newline at end of file
+}
+
+/**
+ *
+ * @param databaseName name of database where to look for table
+ * @param tableName table to look for
+ * @param ownerUri owner uri
+ * @param retryCount number of times to retry with a 5 second wait between each try
+ * Checks for table existing
+ */
+export async function assertTableCreationResult(databaseName: string, tableName: string, ownerUri: string, retryCount: number, checkForData?: boolean): Promise<void> {
+	let result: azdata.SimpleExecuteResult;
+	while (retryCount > 0) {
+		--retryCount;
+		let query = `BEGIN TRY
+				USE ${databaseName}
+				SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_NAME = '${tableName}'
```

Comment:
You should check the schema as well, otherwise dbo.Table1 and mySchema.Table1 would both return true for this